boards/seeed_wio_terminal.json

```diff
@@ -0,0 +1,54 @@
+{
+  "build": {
+    "arduino": {
+        "ldscript": "flash_with_bootloader.ld"
+    },
+    "core": "seeed",
+    "cpu": "cortex-m4",
+    "extra_flags":"-D__SAMD51P19A__ -DWIO_TERMINAL -DSEEED_WIO_TERMINAL -DSEEED_GROVE_UI_WIRELESS -D__SAMD51__ -D__FPU_PRESENT -DARM_MATH_CM4 -mfloat-abi=hard -mfpu=fpv4-sp-d16",
+    "f_cpu": "120000000L",
+    "hwids": [
+      [
+        "0x2886",
+        "0x802D"
+      ],
+      [
+        "0x2886",
+        "0x002D"
+      ]
+    ],
+    "mcu": "samd51p19a",
+    "system": "samd",
+    "usb_product": "Seeed Wio Terminal",
+    "variant": "wio_terminal"
+  },
+  "debug": {
+    "jlink_device": "ATSAMD51P19",
+    "openocd_chipname": "at91samd51p19",
+    "openocd_target": "atsame5x",
+    "svd_path": "ATSAMD51P19A.svd"
```

Comment:
There is no such SVD file in this repo.

Reply:
Should i just remove the `svd path`?

Reply:
It'd be much better if you add this file

Reply:
Not sure where this svd_path actually points to, where do i add the file

Reply:
https://github.com/platformio/platform-atmelsam/tree/develop/misc/svd